packages/flutter/test/material/text_field_test.dart

```diff
@@ -125,12 +125,12 @@ void main() {
   SystemChannels.platform.setMockMethodCallHandler(mockClipboard.handleMethodCall);
 
   const String kThreeLines =
-    'First line of text is '
-    'Second line goes until '
-    'Third line of stuff ';
+    'First line of text is\n'
```

Comment:
I think these lack of new lines in the test data are bugs

Reply:
Did the tests start failing?

I don't think this should be changed. It looks like tests that used kThreeLines set the font size to 34, which would cause the text field to insert soft breaks at the ends of the lines.